configuration/external_parameters.rst

```diff
@@ -114,6 +114,12 @@ of the following:
 
 .. tip::
 
+    .. versionadded:: 3.3
+        The support of the special ``SYMFONY__`` environment variables was
+        deprecated in Symfony 3.3 and it will be removed in 4.0. Instead of
+        using those variables, define regular environment variables and get
+        their values using the ``%env(...)%`` syntax in your config files.
```

Comment:
I have no doubt in French we definitely mean "each" when using "their". To me in this case, each environment variable has one value and not many, I've found many discussions on the web, looks like this confusion is global, see http://english.stackexchange.com/questions/131430/should-a-noun-after-the-determiner-their-take-in-the-form-of-singular-or-of-pl for a reference.